sorc/global_cycle.fd/land_increments.f90

```diff
@@ -420,7 +597,7 @@ subroutine calculate_landinc_mask(smc,swe,vtype,lensfc,veg_type_landice,mask)
 
     ! land (but not land-ice)
     do i=1,lensfc
-        if (smc(i) .LT. 0.99) then
+        if (nint(stype(i)) .GT. 0) then
```

Comment:
what is the soil type of glaciers? (double-checking that they are not being counted as soil).

Reply:
Permanent land ice is indicated by the vegetation type:
https://github.com/ufs-community/UFS_UTILS/blob/develop/sorc/global_cycle.fd/cycle.f90#L478

Reply:
Double-checked. The mask output is 1  - soil, 2 - snow-covered, 0 - land ice, -1  not land.
Land ice is set to 0 in the mask based on the vegetation type as George noted, and hence, land ice is not counted as soil.